### PR TITLE
Updating number validation in reportbacks to not allow 0.

### PR DESCRIPTION
**Pull request by weerd**
@DFurnes Number validation update that you helped me with :tropical_drink:

### CODE REVIEW

**Comment by aaronschachter** [comment]
Is it worth renaming this function to something more generic like `PositiveInteger` vs `ReportbackCount` ?

**Comment by weerd** [comment]
Potentially... Señor @DFurnes can you weigh in since this is your baby? :baby:

**Comment by DFurnes** [comment]
Renaming to `positiveInteger` sounds like a good call to me. :+1:

**Comment by weerd** [comment]
Okie doke. Will do. Updating incoming!

**Comment by DFurnes** [comment]
:shipit:

File: lib/themes/dosomething/paraneue_dosomething/js/validation/reportback.js [hunk 1]
```diff
@@ -4,8 +4,9 @@ define(function (require) {
   var Validation= require("neue/validation");
 
   // validate number of items
-  Validation.registerValidationFunction("reportbackNumber", function(string, done) {
-    if( string !== "" && /^[,0-9]+$/.test(string) ) {
+  Validation.registerValidationFunction("positiveInteger", function(string, done) {
+    var trimmedString = string.replace(" ", "");
+    if( trimmedString !== "" && /^[1-9]\d*$/.test(trimmedString) ) {
```
Comment (aaronschachter) [inline lib/themes/dosomething/paraneue_dosomething/js/validation/reportback.js:9]: Just making sure... (I suck at regex) but this isn't filtering out numbers with 0's in it, is it?  Like if you test with 10 or 101

Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/js/validation/reportback.js:9]: Tested and it's fine :) Only filters out, just `0`, but not integers > 0, with 0's in them :)

**Pull request merged**